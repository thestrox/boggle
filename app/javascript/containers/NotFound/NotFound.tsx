import React from "react";

export default function NotFound() {
    return (
        <div>
            <h1>404</h1>
            <h3>Page not found</h3>
        </div>
    )
}
