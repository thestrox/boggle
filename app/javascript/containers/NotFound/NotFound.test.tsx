import React from 'react';
import { shallow } from 'enzyme';
import NotFound from "./NotFound"

function setup() {
    return {
        component: shallow(<NotFound/>)
    }
}

describe('Tile Component', () => {
    it('should render correctly', () => {
        const {component} = setup();
        expect(component.find('h1').text()).toBe('404');
        expect(component.find('h3').text()).toBe('Page not found');
    });

    it('should test snapshot renders', () => {
        const {component} = setup();
        expect(component).toMatchSnapshot();
    });
})