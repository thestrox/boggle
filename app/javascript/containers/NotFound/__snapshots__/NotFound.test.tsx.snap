// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tile Component should test snapshot renders 1`] = `
<div>
  <h1>
    404
  </h1>
  <h3>
    Page not found
  </h3>
</div>
`;
